src/main/java/com/epam/test_generator/entities/api/JwtTokenBuilder.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.test_generator.entities.api;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTCreator;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.epam.test_generator.entities.User;
+import org.springframework.core.env.Environment;
+
+import java.io.UnsupportedEncodingException;
+
+public class JwtTokenBuilder {
+    public JWTCreator.Builder buildFor(User user, String issuer) {
```

Comment:
To be precise this method cannot be named because it doesn't return built result. It results builder, and therefore must be named *builderFor*